os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/os/tizenrt/osif_tizenrt.c

```diff
@@ -514,22 +530,22 @@ bool osif_msg_send(void *p_handle, void *p_msg, uint32_t wait_ms)
 	int prio = MQ_PRIO_MAX;
 
 	if (!p_handle) {
-		printf("[osif_msg_send] p_handle is NULL\r\n");
+		dbg("p_handle is NULL\r\n");
 		return _FAIL;
 	}
 
-	if (up_interrupt_context() == false && wait_ms != 0xFFFFFFFF) {
+	if (osif_task_context_check() == true && wait_ms != 0xFFFFFFFF) {

```

Comment:
Below is more cheaper condition, but if most of case, wait_ms is  not 0xFFFFFFFF, then keep current style. 
```
(wait_ms != 0xFFFFFFFF && osif_task_context_check() == true)
```

Reply:
Yes, currently most of the is not 0xFFFFFFFF.
We will remain it.